package.json

```diff
@@ -79,4 +81,4 @@
     "url": "https://github.com/electrode-io/electrode-native.git"
   },
   "homepage": "http://www.electrode.io"
-}
+}
```

Comment:
Did a tool remove the final newline here or what happened?

Reply:
No, my bad it would seem. fixed.
prettier is currently configured to ignore json files, otherwise it would have not happened.
don't remember the details as to why we added json files to the exclusion list, but should look into it and open another PR to address this (i.e remove json files from exclusion). Looks like it was done as a quick workaround to some kind of issue (maybe some tests doing exact file content comparison and produced json content is not using prettier rules making test fails if fixture is prettified) but if that's the case we should rather ignore specific json files or directories rather than excluding them all (as first step, and then maybe, if that's the case, apply same prettier rules to generated json files). Will open issue for this based on this comment.